### DISCUSSION TITLE
Add ability to change Accession scoping in the UI

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5771] [Feature Requests]
https://github.com/specify/specify7/blob/9adcc03acd49ad543e5b7742d569436ba743c33a/specifyweb/specify/scoping.py#L24-L29

`IsAccessionsGlobal` is a boolean value in the [Institution table](https://files.specifysoftware.org/schema/version/2.10/#institution) that can be toggled. There should be a more intuitive way in the UI to change this behavior, potentially upon database creation or within the schema configuration tool.

**Comment by maxpatiiuk** [comment]
Is this a duplicate or an extension of #2912?

**Comment by grantfitzsimmons** [comment]
Extension IMO
